mailchimp_ecommerce.module

```diff
@@ -802,8 +802,12 @@ function mailchimp_ecommerce_update_order($order_id, array $order) {
  *   The product SKU.
  * @param float $price
  *   The product price.
+ * @param string $image_url
+ *   A URL to a representative product Image.
+ * @param int $stock
+ *   If enabled, the current active inventory.
  */
-function mailchimp_ecommerce_add_product($product_id, $product_variant_id, $title, $description, $type, $sku, $url, $price, $image_url = '') {
+function mailchimp_ecommerce_add_product($product_id, $product_variant_id, $title, $description, $type, $sku, $url, $price, $image_url = '', $stock) {
```

Comment:
A default value for $stock should be set here since it is optional. `mailchimp_ecommerce_add_product` can and will be called by other modules.

Reply:
Good call. I fixed this in my branch, using `1` for the default seems reasonable. Perhaps we might instead want to create a new constant for this value, e.g. `MAILCHIMP_ECOMMERCE_DEFAULT_INVENTORY_AMOUNT` which we can then map to a default variable (so admin users can override it if they need to, since 1 might not be the best default for everyone). We can do that in a follow-up, as well.